src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef

```diff
@@ -52,4 +52,28 @@
 "1"="{A2BBC114-47E4-473F-A49C-69EE89711243}"
 
 [$RootKey$\OpenFolder\Settings\VSWorkspaceSettings\{EA6637C6-17DF-45B5-A183-0951C54243BC}]
-@="$PackageFolder$\OpenFolderSchema.json"
\ No newline at end of file
+@="$PackageFolder$\OpenFolderSchema.json"
+
+; Registration for DAP version of MIEngine (hosted by OpenDebugAD7)
+[$RootKey$\AD7Metrics\Engine\{F328E937-B412-44AD-905D-90B913C3D098}]
+@="MI Debug Engine"
+"CLSID"="{DAB324E9-7B35-454C-ACA8-F6BB0D5C8673}"
+"Adapter"="$PackageFolder$\vscode\OpenDebugAD7.exe"
+"Language"="C++"
+"LanguageId"="{3A12D0B7-C26C-11D0-B442-00A0244A1DD2}"
+; "AddressBP"=dword:00000001
+"AlwaysLoadLocal"=dword:00000001
+"AlwaysLoadProgramProviderLocal"=dword:00000001
+; "CallStackBP"=dword:00000001
+; "ConditionalBP"=dword:00000001
+; "DataBP"=dword:00000001
+; "Disassembly"=dword:00000001
```

Comment:
Did we want to support Disassembly when in VS?

Reply:
Eventually, but all the necessary requests would need to be implemented in OpenDebugAD7 first.  Out of scope for now.